twilight-lavalink/src/node.rs

```diff
@@ -610,23 +691,42 @@ impl Drop for Connection {
     }
 }
 
+const TWILIGHT_CLIENT_NAME: &str = concat!("twilight-lavalink/", env!("CARGO_PKG_VERSION"));
+
 fn connect_request(state: &NodeConfig) -> Result<ClientBuilder, NodeError> {
     let mut builder = ClientBuilder::new()
-        .uri(&format!("ws://{}", state.address))
+        .uri(&format!("ws://{}/v4/websocket", state.address))
```

Comment:
Since the crate supports TLS, this should use `wss` conditionally. Considering you added TLS support for the HTTP part too, I assume all of that is entirely untested if you left this as-is?

Reply:
see other comment about http2 support. Clarify that and I can then address this.

Reply:
@Gelbpunkt I looked into this a bit more. I was trying to enable tls support but this uri isn't allowed to be tls in the twilight-http. Should we just drop this implementation use and use the hyper client ourselves here?

Reply:
How is twilight-http relevant at all here? This is the websocket connection part. Generally speaking though I think you won't be able to avoid forcing the user to specify whether TLS should be used for a particular node

Reply:
The ClientBuilder is a struct used within the twilight http crate. I don't know why we have the socket connection using this crate and not using the hyper builder directly.

Reply:
No, that's a `tokio-websockets` struct

Reply:
ah sorry for the confusion... missed that. I was using my editor to find the definition and it matched the name of the twilight-http. Didn't double check the scope of it.

Reply:
Alright so here is the deal. I did the rebase but the docs broke and can't reproduce locally. Also I have no idea how to make the TLS happen here. I am trying and running into all sorts of errors.

I get corrupted messages on the twilight side and and a bunch of invalid strings from the lavalink side. It is as if the websocket isn't upgrading to TLS correctly. I can't just simply change from ws to wss is what I am saying. I don't have time to debug this issue. I would like to make this fix later if possible as a issue ticket.

I think the prior behavior of this client was just on the ws not the upgraded TLS socket based on the diff.
`.uri(&format!("ws://{}", state.address))`

How would you like to proceed? I need help if we want this change in this PR.

Reply:
Please add a per-node parameter that toggles the use of TLS